cpp/third_party/CMakeLists.txt

```diff
@@ -8,10 +8,6 @@ if(NOT ${ARCTICDB_USING_CONDA})
     add_subdirectory(rapidcheck/extras/gtest)
     add_subdirectory(Remotery)
 endif()
-add_subdirectory(lmdbcxx)
-add_subdirectory(Remotery)
 
+add_subdirectory(lmdbcxx)
```

Comment:
I thought `lmdbcxx` has also been made available via Conda?

Similarly, the Conda-fy exercise if done correctly should _not_ need to include the `third_party` at all. I.e. instead of doing a `if(NOT ${ARCTICDB_USING_CONDA})`, the directory should be skipped in the parent `CMakeLists.txt` when using Conda.

Reply:
You're right. I missed the PR: https://github.com/man-group/ArcticDB/pull/334

Will remove.